### ISSUE TITLE
Try black pantyhose with polka dots for training

### ISSUE THREAD

**Issue by zhaochi12** [rohitgandikota/sliders#54]
I'm wondering if I understood it wrong....

I want to train a SD1.5 black silk LORA with wavy points through Visual Concept Sliders. I have prepared two sets of image data sets, each set of 14 photos, with a resolution of 3024*4032, and the names are 1 to 14. Photos One-to-one correspondence. One data set is multi-angle photos without clothes, and the other data set is multi-angle photos wearing only black polka-dot pantyhose.
However, there is almost no change when loading LORA after training.

This is how I set it up:
prompts_file: "F:/sliders-for-windows-main/sliders-for-windows-main/trainscripts/imagesliders/data/prompts.yaml"

pretrained_model:
   name_or_path: "F:/sd-webui-aki/sd-webui-aki-v4.1/sd-webui-aki-v4.1/models/Stable-diffusion/revAnimated_v122.safetensors" # you can also use .ckpt or .safetensorsmodels

   v2: false # true if model is v2.x

   v_pred: false # true if model uses v-prediction

network:
   type: "c3lier" # or "c3lier" or "lierla"
   rank: 16
   alpha: 1.0
   training_method: "noxattn"

train:
   precision: "bfloat16"
   noise_scheduler: "ddim" # or "ddpm", "lms", "euler_a"
   iterations: 2000
   lr: 0.00002
   optimizer: "AdamW"
   lr_scheduler: "constant"
   max_denoising_steps: 50

save:
   name: "tempbd_v5"
   path: "F:/sliders-for-windows-main/sliders-for-windows-main/models"
   per_steps: 400
   precision: "bfloat16"
logging:
   use_wandb: false
   verbose: false
other:
   use_xformers: true

- target: "polka dot pantyhose" # what word for erasing the positive concept from
   positive: "polka dot pantyhose,pantyhose,black pantyhose,white pantyhose" # concept to erase
   unconditional: "worst quality, low quality, normal quality" # word to take the difference from the positive concept
   neutral: "pantyhose" # starting point for conditioning the target
   action: "enhance" # erase or enhance
   guidance_scale: 4
   resolution: 1024
   dynamic_resolution: true
   batch_size: 1

In addition, it is recommended to add ims = [im_ for im_ in ims if '.png' in im_ or '.jpg' in im_ or '.jpeg' in im_ or '.webp' in im_ or '.JPG' in im_ ]. I had a bug because the photo was a JPG with capital letters. I couldn't recognize the photo. I spent a long time debugging it...
In addition, I added TensorBoard to the program and found that the loss was very messy. I don't know if this is normal...
<img width="1222" alt="捕获" src="https://github.com/rohitgandikota/sliders/assets/5520626/8a73916e-5853-4374-ae13-e99cd22e6cc1">

**Comment by rohitgandikota** [comment]
I would recommend not adding any prompts for target. My suggested prompts
```
target: "" # what word for erasing the positive concept from
positive: "polka dot pantyhose,pantyhose,black pantyhose,white pantyhose" # concept to erase
unconditional: "" # word to take the difference from the positive concept
neutral: "" # starting point for conditioning the target
action: "enhance" # erase or enhance
guidance_scale: 4
resolution: 1024
dynamic_resolution: true
batch_size: 1
```

**Comment by zhaochi12** [comment]
> I would recommend not adding any prompts for target. My suggested prompts
> 
> ```
> target: "" # what word for erasing the positive concept from
> positive: "polka dot pantyhose,pantyhose,black pantyhose,white pantyhose" # concept to erase
> unconditional: "" # word to take the difference from the positive concept
> neutral: "" # starting point for conditioning the target
> action: "enhance" # erase or enhance
> guidance_scale: 4
> resolution: 1024
> dynamic_resolution: true
> batch_size: 1
> ```

Thank you very much, I will try again!~~

**Comment by zhaochi12** [comment]
> I would recommend not adding any prompts for target. My suggested prompts
> 
> ```
> target: "" # what word for erasing the positive concept from
> positive: "polka dot pantyhose,pantyhose,black pantyhose,white pantyhose" # concept to erase
> unconditional: "" # word to take the difference from the positive concept
> neutral: "" # starting point for conditioning the target
> action: "enhance" # erase or enhance
> guidance_scale: 4
> resolution: 1024
> dynamic_resolution: true
> batch_size: 1
> ```

Hello, after I modified it according to your parameters, it still has no effect.... Is there anything else that can be adjusted or modified?

**Comment by rohitgandikota** [comment]
It could also be some inference time issue - could you try looking at the `start_noise` setting? If not, maybe also try text slider ?

**Comment by zhaochi12** [comment]
> It could also be some inference time issue - could you try looking at the `start_noise` setting? If not, maybe also try text slider ?

Okay, then I'll try them all...

**Comment by rohitgandikota** [comment]
closing the issue. Please feel free to reopen if issues persist. Thanks!

**Comment by Linaghan34** [comment]
> 我想知道我是不是理解错了......
> 
> 我想通过视觉概念滑块训练带有波浪点的 SD1.5 黑丝 LORA。我准备了两组图像数据集，每组14张照片，分辨率为3024*4032，名称为1到14。照片 一对一通信。一个数据集是不穿衣服的多角度照片，另一个数据集是只穿着黑色波点连裤袜的多角度照片。但是，训练后加载 LORA 时几乎没有变化.
> 
> 这是我的设置方式： prompts_file：“F：/sliders-for-windows-main/sliders-for-windows-main/trainscripts/imagesliders/data/prompts.yaml”
> 
> pretrained_model： name_or_path： “F：/sd-webui-aki/sd-webui-aki-v4.1/sd-webui-aki-v4.1/models/Stable-diffusion/revAnimated_v122.safetensors” # 也可以使用 .ckpt 或 .safetensorsmodels
> 
> v2： false # 如果模型为 v2.x 则为 true
> 
> v_pred： false # true 如果模型使用 V 预测
> 
> 网络： 类型： “C3lier” # 或 “C3lier” 或 “Lierla” 等级： 16 alpha： 1.0 training_method： “Noxattn”
> 
> train： precision： “bfloat16” noise_scheduler： “ddim” # or “ddpm”， “lms”， “euler_a” 迭代次数： 2000 LR： 0.00002 优化器： “AdamW” lr_scheduler： “常量” max_denoising_steps： 50
> 
> 保存： 名称： “tempbd_v5” 路径： “F：/sliders-for-windows-main/sliders-for-windows-main/models” per_steps： 400 精度： “bfloat16” 日志记录： use_wandb： false 详细： false 其他： use_xformers： true
> 
> * 目标：“波点连裤袜” # 从积极中抹去积极概念的词：“波点连裤袜，连裤袜，黑色连裤袜，白色连裤袜” # 无条件抹去的概念：“最差质量，低质量，正常质量” # 从积极概念中取出区别的词：“连裤袜” # 调节目标动作的起点：“增强” # 擦除或增强 guidance_scale：4 分辨率： 1024 dynamic_resolution：真batch_size：1
> 
> 此外，如果 im_ 中的“.png”或 im_ 中的“.jpg”或 im_ 中的“.jpeg”或 im_ 中的“.webp”或“”，则建议在 ims 中添加 ims = [im_ for im_。JPG“im_]。我有一个错误，因为照片是大写字母的JPG。我认不出这张照片。我花了很长时间调试它......另外，我在程序中添加了 TensorBoard，发现损失非常混乱。我不知道这是否正常...... <img alt="捕获" width="1222" src="https://private-user-images.githubusercontent.com/5520626/292636562-8a73916e-5853-4374-ae13-e99cd22e6cc1.PNG?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTQyMzIyODEsIm5iZiI6MTcxNDIzMTk4MSwicGF0aCI6Ii81NTIwNjI2LzI5MjYzNjU2Mi04YTczOTE2ZS01ODUzLTQzNzQtYWUxMy1lOTljZDIyZTZjYzEuUE5HP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDQyNyUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDA0MjdUMTUzMzAxWiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9MGU3ZDQ0ZGE2OTU5OGExN2Y4MmY0NTNhNDZhZjU0OWNmYjBjZWM0MzA5ZDVmOGFmNWM4ZjYxNTUzM2QwZTYxYiZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.IZ37_vz8jwJSzTmrnnJGbQbi7binUbT5RD9EScEAjlg">

After installing tensorboard in miniconda, tensorboard --logdir=<directory_name>, where is the logs folder called, there is only a preset json file in the logs folder.After installing tensorboard in miniconda, tensorboard --logdir=<directory_name>, where is the logs folder called, there is only a preset json file in the logs folder.
